src/plugins/data_source_management/public/components/direct_query_data_sources_components/direct_query_sync/direct_query_sync_banner.tsx

```diff
@@ -53,14 +59,28 @@ export const DashboardDirectQuerySyncBanner: React.FC<DirectQuerySyncProps> = ({
     };
 
     loadSyncInfo();
+    const interval = setInterval(loadSyncInfo, 10000);
```

Comment:
nitpick: is 10k the default length it takes? https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/DEVELOPER_GUIDE.md#avoid-magic-numbersstrings

Reply:
10 seconds is a convenient-enough refresh, exact value doesn't matter -- happy to do a different number if it's better, but 10s seems lightweight enough while being decently responsive. I'll push a commit to remove the magic.